lib/internal/test_runner/reporter/utils.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const symbols = {
```

Comment:
I am not sure how I feel about this name 🤔 
WDYT about `reporterUnicodeSymbolMap` etc?

Reply:
We could also export them directly or would that be an issue with prototypes?

Reply:
It is not an issue... we need this when creating an object, not a string (so `module.exports.something = '\u2176'` is just fine 🤷🏽‍♂️ 
But I don't think we should export them directly, as we would want this file to have additional common code for the reporters, see https://github.com/nodejs/node/pull/52655#discussion_r1577259028